layouts/partials/head.html

```diff
@@ -1,3 +1,6 @@
+<!-- The head.html partials from old K8S and Docsy have been merged here. Some script calls
+from K8S are removed, e.g. Bootstrap, as they are already brought in with Docsy -->
```

Comment:
nit: this comment might not make much sense once the switch to Docsy has happened